src/static/images/bulb.svg

```diff
@@ -3,9 +3,9 @@
-                    <path fill="#335280" d="M128.852 126.641l12.14 12.21c.201.197.315.466.315.747 0 .282-.114.551-.315.748-.213.176-.486.262-.762.24-.279-.007-.545-.117-.747-.31l-12.125-12.14c-.388-.417-.377-1.066.026-1.469.402-.402 1.052-.414 1.468-.026zm-91.275.097c.402.402.414 1.052.025 1.468l-12.139 12.055c-.197.201-.465.317-.747.324-.281-.01-.548-.125-.747-.324-.36-.401-.36-1.008 0-1.41l12.139-12.14c.417-.387 1.066-.376 1.469.027zm126.213-45.44c.584 0 1.058.474 1.058 1.058 0 .584-.474 1.057-1.058 1.057h-17.159c-.584 0-1.057-.473-1.057-1.057s.473-1.058 1.057-1.058h17.16zm-145.461 0c.584 0 1.057.474 1.057 1.058 0 .584-.473 1.057-1.057 1.057H1.213c-.584 0-1.058-.473-1.058-1.057s.474-1.058 1.058-1.058h17.116zm122.649-57.087c.201.196.315.466.315.747 0 .282-.114.551-.315.747l-12.126 12.14c-.197.2-.466.311-.747.31-.28.001-.55-.11-.747-.31-.412-.413-.412-1.082 0-1.495l12.125-12.14c.413-.411 1.082-.411 1.495 0zm-115.515 0l12.14 12.14c.412.412.412 1.08 0 1.494-.198.2-.467.311-.748.31-.28.003-.55-.11-.747-.31l-12.14-12.14c-.388-.416-.376-1.066.027-1.468.402-.403 1.051-.414 1.468-.026zM82.48.045c.584 0 1.058.473 1.058 1.057v17.159c-.008.58-.477 1.05-1.058 1.057-.584 0-1.057-.473-1.057-1.057V1.102c0-.584.473-1.057 1.057-1.057z" transform="translate(-1313 -155) translate(538 48) translate(775 107)"/>
+                    <path fill="#003B4E" d="M128.852 126.641l12.14 12.21c.201.197.315.466.315.747 0 .282-.114.551-.315.748-.213.176-.486.262-.762.24-.279-.007-.545-.117-.747-.31l-12.125-12.14c-.388-.417-.377-1.066.026-1.469.402-.402 1.052-.414 1.468-.026zm-91.275.097c.402.402.414 1.052.025 1.468l-12.139 12.055c-.197.201-.465.317-.747.324-.281-.01-.548-.125-.747-.324-.36-.401-.36-1.008 0-1.41l12.139-12.14c.417-.387 1.066-.376 1.469.027zm126.213-45.44c.584 0 1.058.474 1.058 1.058 0 .584-.474 1.057-1.058 1.057h-17.159c-.584 0-1.057-.473-1.057-1.057s.473-1.058 1.057-1.058h17.16zm-145.461 0c.584 0 1.057.474 1.057 1.058 0 .584-.473 1.057-1.057 1.057H1.213c-.584 0-1.058-.473-1.058-1.057s.474-1.058 1.058-1.058h17.116zm122.649-57.087c.201.196.315.466.315.747 0 .282-.114.551-.315.747l-12.126 12.14c-.197.2-.466.311-.747.31-.28.001-.55-.11-.747-.31-.412-.413-.412-1.082 0-1.495l12.125-12.14c.413-.411 1.082-.411 1.495 0zm-115.515 0l12.14 12.14c.412.412.412 1.08 0 1.494-.198.2-.467.311-.748.31-.28.003-.55-.11-.747-.31l-12.14-12.14c-.388-.416-.376-1.066.027-1.468.402-.403 1.051-.414 1.468-.026zM82.48.045c.584 0 1.058.473 1.058 1.057v17.159c-.008.58-.477 1.05-1.058 1.057-.584 0-1.057-.473-1.057-1.057V1.102c0-.584.473-1.057 1.057-1.057z" transform="translate(-1313 -155) translate(538 48) translate(775 107)"/>
```

Comment:
Can you optimise this SVG please? 👉🏻 https://jakearchibald.github.io/svgomg/

Reply:
I always thought that CRA auto optimizes them, but I guess not! Optimized!